syncstorage-db/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+//! Generic db abstration.
+
+#[cfg(test)]
+#[macro_use]
+extern crate slog_scope;
+
+pub mod mock;
+#[cfg(test)]
+mod tests;
+
+use std::time::Duration;
+
+use cadence::{Gauged, StatsdClient};
+use tokio::{self, time};
+
+#[cfg(feature = "mysql")]
+pub type DbPool = syncstorage_mysql::MysqlDbPool;
+#[cfg(feature = "mysql")]
+pub use syncstorage_mysql::DbError;
+#[cfg(feature = "mysql")]
+pub type Db = syncstorage_mysql::MysqlDb;
+
+#[cfg(feature = "spanner")]
+pub type DbPool = syncstorage_spanner::SpannerDbPool;
+#[cfg(feature = "spanner")]
+pub use syncstorage_spanner::DbError;
+#[cfg(feature = "spanner")]
+pub type Db = syncstorage_spanner::SpannerDb;
+
+pub use syncserver_db_common::{GetPoolState, PoolState};
+pub use syncstorage_db_common::error::DbErrorIntrospect;
+
+pub use syncstorage_db_common::{
+    params, results,
+    util::{to_rfc3339, SyncTimestamp},
+    DbPoolTrait, DbTrait, Sorting, UserIdentifier,
+};
```

Comment:
I re-export a bunch of stuff here so the syncserver crate only needs to deal with a simple, flat interface